### DISCUSSION TITLE
中壢交通好氣人

### DISCUSSION THREAD

**Discussion by harrycoconut** [1101-GS4719-C/Python-Community-GS4719-C#1289] [General]
今天開車去買甜甜圈的路上，遇到了一位路怒症患者，在紅燈一轉綠的時候開始瘋狂按喇叭📣當時在車裡的我心裡很不是滋味，超想在他車上亂塗鴉😡但後來想想，在中央大學騎車或開車的時候好像常常遇到這些路怒症情況，會不會是因為學生的開車技術比較不純熟呢？亦或是中壢的的卡車司機特別多（多數都有路怒症😌）？
歡迎大家跟我分享你們的路怒症經驗～

**Comment by Tzuhsuannnnn** [comment]
中壢超多人都不喜歡打方向燈:(

**Comment by CPW0919** [comment]
中壢的交通真的很危險，下山的路上好多中老年人騎車都好可怕，還有很多猴子不戴安全帽亂，真想撞他們

**Reply by chia-kai-chang** [reply]
萬萬不可阿

**Comment by tower20021202** [comment]
中壢三寶真的很多，而且學校後門跟宵夜街真的很多人連騎車都不戴安全帽，哪天出事撞到頭真的是剛好而已

**Comment by CCello** [comment]
中壢其他地方我不確定，但學校後門的交通是真的很差，尤其是中午時段常常可以看到大型貨車在卸貨，導致整條路阻塞，而且有些駕駛都不會禮讓行人，十分危險。

**Comment by brian431** [comment]
雖然跟本文沒什麼關係。近一個月netflix有一齣蠻紅的劇叫怒嗆人生，故事就是由路怒症事件所引發的一連串曲折離奇的事件。還蠻有趣的，推薦大家去看看。其中不只好笑，也探討了許多議題。

**Comment by Jack-1D** [comment]
感覺是駕駛的平均素質太低，駕照太容易取得，和交通規劃太差的問題。如果路上的車少一點，大家能試著體諒彼此，我想也不會有這麼多行車糾紛，台灣也不會變成行人地獄。

**Comment by Superlan8** [comment]
身為土生土長的台南人，我對這種交通亂象早已見怪不怪，台南三寶跟圓環也是出名的呢

**Comment by YoYooHuang** [comment]
我是台南來的，在台灣這種情況已經見怪不怪了。與其改變他們，不如好好保護自己，在路上騎車多注意路況，前後左右都要小心。前幾天就看到有汽車要右轉，差點撞到左方疾駛而來的機車，連我都替他們擔心了。希望大家都能快樂出門，平安回家！

**Comment by orianna901001** [comment]
我大一的那年，因為前方阿伯鬼切的左右轉燈，直接撞到宵夜街上坡的山面，而且那邊的坡還讓我倒在那邊起不來QQ
(幸好路過的foodpanda來幫我拉車)

**Comment by ALAN30359704** [comment]
超有同感！尤其是在中壢火車站那邊的人行道超危險！公車停靠的位置擋到人的視線，過馬路很難知道轉彎處有沒有車！

**Comment by llh26** [comment]
真的希望大家都能重視行車安全，曾經看過一個影片：在日本，行車一定會禮讓行人，所以當一個台灣人去到日本的時候，就會發現車子和行人誰也不敢動，因為在台灣都是行人在禮讓車輛的⋯。政府做的那些標語完完全全被那些人拋在腦後，「只把自己的便利放第一」，完全沒在care別人的生命安全，開車真的要小心，到處都可能有三寶出沒。

**Comment by Jamesmoneya** [comment]
我覺得可以把範圍縮小到中央的三寶好多，我覺得以學小的環境來說，根本不該開放車子進去校園，校園的人行道也根本不夠下課時間學生的人潮，因此在尖峰時間會有人、腳踏車、公車、小客車、貨車、根外送機車，同時在一線道上，這樣開放車子進校園根本罔顧學生的安全，也沒有良好的人行道環境
